### DISCUSSION TITLE
Export & Ressources

### DISCUSSION THREAD

**Discussion by slimooo** [retypeapp/retype#23] [Q&A] [answered]
Hello,

I have a problem when I export. I have a wrong path for resssources.

Example : https://slimooo.github.io/dpo/

**Comment by geoffreymcgill** [comment, marked as answer]
Hi @slimooo, 

You likely need to set `"base": "dpo"` in your `retype.json`. It's a requirement to support GitHub Pages if you're using the default `github.io` domain. If you configure a `Custom domain` in GitHub Pages, then the `base` config is not required.

Here's some relevant docs:

https://retype.com/guides/github_actions/#add-base-if-github-io

https://retype.com/configuration/project/#base

Unfortunately, we cannot figure out a way to automate setting that value.

**Reply by geoffreymcgill** [reply]
ah, maybe that won't fix the issue for you. I just looked at your GitHub repo and you're committing the raw .html files.

The `base` suggestion above would be required if using the [Retype GitHub Action](https://retype.com/guides/github_actions/).

I'll try and figure out what's going wrong with your setup. One moment...

**Reply by geoffreymcgill** [reply]
okay, it looks like you might actually have the `base` configured, but shouldn't.

In your project `retype.json`, I believe you have the following `base` configured:

```json
{
  "base": "france"
}
```

but, you can remove that `base` config.

**Reply by geoffreymcgill** [reply]
nope, you need `"base": "dpo"` in your `retype.json`. 

```json
{
  "base": "dpo"
}
```

Sorry for the confusion.

**Reply by geoffreymcgill** [reply]
Looks like you got it up and running. Nice.

**Reply by slimooo** [reply]
Yes thank you, that's work :) Really good job! Thank you so much

**Comment by geoffreymcgill** [comment]
If you're hosting with GitHub Pages, I'd recommend using the Retype GitHub Action to automate the building of the website. 

https://retype.com/guides/github_actions/

Then you just need to edit your `.md` pages and the action will build and deploy the website automatically on every new commit to the repo.

Let me know if you need any assistance setting up the Retype GitHub Action or configuring GitHub Pages and I'll be happy to help. Feel free to use the online chat on the https://retype.com website too.
